pkg/sysinfo/sysinfo_linux.go

```diff
@@ -33,6 +33,11 @@ func New(quiet bool) *SysInfo {
 		if !sysInfo.OomKillDisable && !quiet {
 			logrus.Warnf("Your kernel does not support oom control.")
 		}
+		_, err = ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.swappiness"))
```

Comment:
@cpuguy83 since mem-swappiness is not critical shall I change such that this does not make bailing out of container creation ?
so that atleast we know that swappiness did not work from warning..?
something like (similar to swap memory):
 _, err1 = ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.swappiness"))
                sysInfo.MemSwappiness = err1 == nil
                if !sysInfo.MemSwappiness && !quiet {
                        logrus.Warnf("Your kernel does not support container swappiness control.")
                }
OR should I remove whole check?